components/src/dynamo/replay/main.py

```
@@ -508,7 +522,23 @@ def _run_planner_replay(
                             f"(prefill={len(prefill_fpms)}, decode={len(decode_fpms)})\n"
                         )
 
-    return adapter.run()
+    result = adapter.run()
+
+    # Surface true GPU-hours in the JSON report: the time-integrated provisioned
+    # worker-seconds (which already include the startup ramp and drain tail) ×
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)